tests/functional/builtins/codegen/test_raw_call.py

```diff
@@ -614,6 +614,19 @@ def foo(_addr: address):
     (
         """
 @external
+def foo(a: address):
```

Comment:
I added a test for raw call because it has its own analysis in its `build_IR` function. Can I move these syntax tests to `tests/functional/syntax/test_raw_call.py`?

Reply:
yea

Reply:
done!